frontend/src/components/managePermissions/table/ApplicationAdminTable.vue

```diff
@@ -91,7 +84,7 @@ const deleteAdmin = (admin: FamAppAdminGetResponse) => {
         <div class="custom-data-table">
             <DataTableHeader
                 btnLabel="Add application admin"
-                :btnRoute="routeItems.grantUserPermission.path"
+                :btnRoute="routeItems.grantAppAdmin.path"
```

Comment:
Hello, Nick!
I think you forgot to add this path to the side nav as well

Reply:
Nice find! Thanks, Jota!